libs/auth/src/lib/auth.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { TestBed } from '@angular/core/testing'
+import { AuthService } from './auth.service'
+import { MeApiService } from '@lib/gn-api'
+import { Subject } from 'rxjs'
+
+describe('AuthService', () => {
+  let service: AuthService
+  let meApiMock
+  let me$
+
+  beforeEach(() => {
+    me$ = new Subject()
+    meApiMock = {
+      getMe: () => me$,
+    }
+
+    TestBed.configureTestingModule({
```

Comment:
you should not use `TestBed` for services, there are simple classes and should be instanciated with `new`, specially when all injection dependices are mocks.

It's way faster to execute tests this way.